package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "next lint",
     "db:generate": "prisma migrate dev --name $1",
     "db:migrate": "prisma migrate deploy",
-    "db:seed": "prisma db seed",
+    "db:seed":  "ts-node --files --compiler-options '{\"module\":\"commonjs\"}' src/backend/scripts/seed.ts",
```

Comment:
With this change `npx prisma seed` is still gonna be broken. Prisma documentation says that you need to add `prisma.seed` key to package.json

https://www.prisma.io/docs/guides/migrate/seed-database

Here's an example from issues https://github.com/taskany-inc/issues/blob/main/package.json#L23

Reply:
thanks!